code/script.js

```diff
@@ -29,23 +83,358 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+ 
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
+  
   showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+
+}
+
+
+btn.addEventListener("click" , (e) => {
+  e.preventDefault();
+
+  const enterName = nameInput.value.trim();
+  
+  if(enterName !== ""){
+    message = nameInput.value;
+    showMessage(enterName , 'user');
+    clearInput();
+    setTimeout(() => {
+      showMessage(`Nice to meet you ${enterName}.what type of food would you like to order?` , 'bot')
+      btnsEl.style.display = "flex";
+      nameFormEl.style.display = "none";
+    },1000);
+
+  }else{
+    alert("Inavlid Name , please enter your name")
+  }
+ 
+
+});
+
+const clearInput = () =>{
+  nameInput.value = "";
 }
 
-// Set up your eventlisteners here
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
+
+let selectedFood = "";
+let selectedSide = "";
+
+pizzaBtn.addEventListener("click" , () =>{
+  showMessage(`Pizza` , 'user');
+  btnsEl.style.display = "none";
+  setTimeout(()=>{
+    showMessage(`oh so you are in the mood for Pizza? Great choice! please select something from menue `, 'bot')
+    pizzaMenue.style.display = "flex";
+    pastaMenue.style.display = "none";
+    saladMenue.style.display = "none";
+  },1000);
+
+  pizzaMenue.addEventListener("change" , ()=>{
+    let valueOptionPizza = pizzaMenue.value;
+
+    if(valueOptionPizza === "Margarita"){
+      showMessage("Margarita" , 'user');
+      setTimeout(() =>{
+        showMessage("please choose how many Margarita Pizaa would you like?" , 'bot');
+        addBtns.style.display = "flex";
+        selectedFood = "Margarita Pizaa";
+        price = pizzaPrice.Margarita
+
+      },1000)
```

Comment:
This is so interesting, adding the 1000 at the end, never would have thought, cool that it works like that!

Reply:
Thank you for your review and comment Ulrika